scripts/install-mender.sh

```diff
@@ -218,6 +218,74 @@ print_components() {
     done
 }
 
+
+#
+# $1 - component
+# $2 - mender-release
+#
+# Note, requires jq and curl
+function get_version_of() {
+    which jq 2>&1 >/dev/null || { echo >&2 "'jq' needs to be installed"; exit 1; }
+    [[ $# -ne 2 ]] && { echo >&2 " $# get_version_of requires two arguments"; exit 1; }
+    local -r component_name="$1"
+    local -r mender_release="$2"
+    local -r major_minor="$(echo -e ${mender_release} | cut --delimiter=. --fields=1,2)"
+    local -r versions_file="$(curl --fail https://docs.mender.io/releases/versions.json)"
+    echo "$versions_file" | jq --raw-output "$(cat <<EOF
+.releases |
+."${major_minor}" |
+."${mender_release}" |
+.repos |
+.[] |
+ select(.name == "${component_name}") |
+.version
+EOF
+)"
+}
+
+#
+# Returns the latest Mender LTS release version
+# from `docs.mender.io/`
+#
+# Note, requires jq and curl
+function get_latest_mender_lts_release() {
+    [[ $# -ne 0 ]] && { echo >&2 "get_latest_mender_lts_release takes no arguments"; exit 1; }
+    local -r versions_file="$(curl --fail https://docs.mender.io/releases/versions.json)"
+    local -r lts="$(echo "${versions_file}" | jq '.lts[0]')"
+    local -r latest_lts="$(echo "${versions_file}" | jq --raw-output "$(cat <<EOF
+.releases[] |
+ to_entries[].key |
+ select(startswith($lts))
+EOF
+)" |
+head --lines=1
+)"
+    echo ${latest_lts}
+}
+
+#
+# $1 - The package to install
+#
```

Comment:
I would add a note that this is meant only for commercial components. Otherwise the `case` below looks incomplete.

Reply:
Renamed it to reflect this